client/go/outline/vpn_linux.go

```diff
@@ -27,6 +27,7 @@ import (
 type vpnConfigJSON struct {
 	VPNConfig       vpn.Config `json:"vpn"`
 	TransportConfig string     `json:"transport"`
+	SessionConfig   string     `json:"session_report"`
```

Comment:
The whole `vpnConfigJSON` is the session config
```suggestion
	ReportConfig   string     `json:"report"`
```